e2e/scaffolding/ethereum.ts

```diff
@@ -55,6 +55,14 @@ export function getUnifiedPublicKey(pair: KeyringPair): Uint8Array {
   return pair.publicKey;
 }
 
+export function reverseUnifiedAddressToEthereumAddress(unifiedAddress: Uint8Array) : string {
+  const hex = Buffer.from(unifiedAddress).toString("hex");
+  if (!hex.toLowerCase().endsWith("eeeeeeeeeeeeeeeeeeeeeeee")) {
```

Comment:
nit: might be more readable as `'ee'.repeat(12)` to make it clear that we're looking for 12 bytes of 0xEE.
```suggestion
  if (!hex.toLowerCase().endsWith("ee".repeat(12))) {
```